setup.py

```diff
@@ -60,35 +59,6 @@
     'cherrypy.scaffold',
     'cherrypy.wsgiserver',
 ]
-data_files = [
-    ('cherrypy', [
-        'cherrypy/cherryd',
-        'cherrypy/favicon.ico',
```

Comment:
I didn't check helloworld using your branch, I just pointed that your previous check was irrelevant :)